tests/integration/cmd_namespace_test.go

```diff
@@ -183,6 +185,18 @@ ComponentSettings:
 					return out
 				}, 10*time.Second, 1*time.Second).Should(ContainSubstring(commonVar.Project))
 			})
+			It(fmt.Sprintf("should successfully list all the %ss in JSON format", commandName), func() {
+				Eventually(func() bool {
+					ns := helper.Cmd("odo", "list", commandName).ShouldPass().Out()
+					return strings.Contains(ns, commonVar.Project)
+				}).WithTimeout(10 * time.Second).Should(BeTrue())
+
+				out := helper.Cmd("odo", "list", commandName, "-o", "json").ShouldPass().Out()
+				Expect(helper.IsJSON(out)).To(BeTrue())
+				// check if the namespace/project created for this test is marked as active in the JSON output
+				gjsonStr := fmt.Sprintf("namespaces.#[name==%s].active", commonVar.Project)
+				Expect(gjson.Get(out, gjsonStr).String()).To(Equal("true"))
```

Comment:
Besides that, how about also making sure the current namespace/project is not listed in the non-active ones?

```go
Expect(gjson.Get(out, "namespaces.#[active==false]#.name").String()).ShouldNot(ContainSubstring(commonVar.Project))
```

Reply:
IMO, if you check that there is a `{ "name": "ns", "active": true }` (the first check), you cannot have a `{ "name": "ns", "active": false }` (the second check)

Reply:
I agree. I actually wanted us to make sure that any other projects/namespaces in the output do have the `active` field set to `false`.

Reply:
Depending upon when the tests are running, the output could contain multiple projects that are marked `active: false`. So while it might make sense, I don't think it will be accurate, unless we check every namespace marked as `active: false`.

Reply:
ok, I misunderstood the query, that would be interesting to test, effectively

Reply:
When I replied, I could see only the top comment. GH doing tricks. 
OK, I have added a check for that. On CI, we don't expect to be in a scenario where there won't be any other namespaces, so this wouldn't be a problem.